### DISCUSSION TITLE
Add google maps type control

### DISCUSSION THREAD

**Discussion by Lazaaa** [elementor/elementor#22876] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In many places, it would be necessary for the Google Maps embed to be displayed not only in Roadmap mode.

### Describe the solution you'd like

 It would be nice if the following modification was included in the next version.
 google-maps.php
```
$this->add_control(
	'view',
	[
		'label' => esc_html__( 'View', 'elementor' ),
		'type' => Controls_Manager::HIDDEN,
		'default' => 'traditional',
	]
);
	
$this->add_control(
    'map_type_control',
    [
        'label' => esc_html__( 'Map Type', 'elementor' ),
        'type' => Controls_Manager::SELECT,
        'options' => [
            'roadmap' => __( 'Roadmap', 'elementor' ),
            'satellite' => __( 'Satellite', 'elementor' ),
            'hybrid' => __( 'Hybrid', 'elementor' ),
            'terrain' => __( 'Terrain', 'elementor' ),
        ],
        'default' => 'roadmap',
    ]
);

//....

if ( $api_key ) {
    $params[] = $api_key;

    $url = 'https://www.google.com/maps/embed/v1/place?key=%3$s&q=%1$s&amp;zoom=%2$d&amp;maptype=%4$s';
} else {
    $url = 'https://maps.google.com/maps?q=%1$s&amp;t=m&amp;z=%2$d&amp;output=embed&amp;iwloc=near&amp;maptype=%3$s';
}

$params[] = $settings['map_type_control'];
```

**Comment by matt-bml** [comment]
I strongly second this request.  Or some alternative method to specify the map type with the Google Maps widget.  It seems to be defaulting to satellite view on our website these days, which is definitely not what we want.  I'd like to be able to specify roadmap in my particular case, but being able to choose would be best.

While I could embed a Google Map myself, none of the other site maintainers at my org would be able to replicate or adjust that.  We'd much prefer to keep using Elementor widgets everywhere we can.

Apologies if there is a way I was unable to find.  I'd be very glad to be told of one!

**Reply by matt-bml** [reply]
> It seems to be defaulting to satellite view on our website these days, which is definitely not what we want.

Oops, I take that back, partially.  With a bit more testing, it seems to be unpredictable which view is shown.  At least, I haven't figured out a pattern yet.  The last several times I've visited our "Visit" page, I have been shown the roadmap view.  But earlier today, I was definitely getting satellite.

Regardless, it would be a great additional feature to be able to specify map type in the widget.
